Source/WebKit/Shared/WebPreferencesDefaultValues.cpp

```diff
@@ -54,6 +54,12 @@ bool defaultCSSOMViewScrollingAPIEnabled()
     return !result;
 }
 
+bool defaultShouldPrintBackgrounds()
+{
+    static bool result = !linkedOnOrAfterSDKWithBehavior(SDKAlignedBehavior::DefaultsToExcludingBackgroundsWhenPrinting);
```

Comment:
Someday we should convince Sam to make this a thing you can do straight from the YAML (since I see a bunch of them).